Project1/Menu.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Project1
+{
+    internal class Menu
+    {
+        public void ShowMainMenu()
+        {
+            Console.WriteLine("1. Prosty kalkulator");
+            Console.WriteLine("2. Zaawansowany kalkulator");
+            Console.WriteLine("3. Działania na plikach txt");
+        }
+        public void ShowBasicMain()
```

Comment:
Tu chyba powinno być ShowBasicMenu

Reply:
A bardziej bym nazwał ShowBaseCalculatorMenu bo bardziej mi to mówi co to ma być. W twoim przypadku mówi to " Pokaż podstawowe menu" czyli jakby było podstawowe i rozszerzone menu a nam zalezy na podstawowym i zaawansowanym kalkulatorze a nie menu